pyproject.toml

```diff
@@ -30,7 +30,6 @@ pytest-asyncio = "^0.24.0"
 [tool.poetry.group.dev.dependencies]
 black = ">=24.8.0,<25.0.0"
 flake8 = ">=7.1.1,<8.0.0"
-openai = ">=1.35.12,<2.0.0"
```

Comment:
I figured out why openai is in the devdependencies, it's because it is used in the mainguards to test out classes like the BaseAgent, so it should remain here or they will break